docs/pages/getting-started.md

```diff
@@ -0,0 +1,39 @@
+# Getting started
+When you have installed all the [prerequisites](/prerequisites.md) you are good to go.
+
+Start by installing the Simple Fable template:
+```
+dotnet new --install "Fable.Template::*"
+```
+Create a directory for your project:
+```
+mkdir FableApp
+cd FableApp
+```
+Scaffold a Fable app from the template:
+```
+dotnet new Fable
+```
+This will create Fable project in this directory. Now install the npm dependencies using yarn:
+```
+yarn install
+```
+Install Fable and dotnet dependencies using Paket:
+```
+dotnet restore
+```
+At this point, when you navigate through your code using Visual Studio Code or Atom with the Ionide extension, you should have tooltips and auto-completion working. 
+
+To run the app in developement mode, you run:
+```
+dotnet fable npm-run start
+```
+This will run two servers in parallel: the Fable compilation server and the webpack development server.
```

Comment:
To avoid some confusion as @mizzle-mo had, I would mention explicitly that `dotnet fable npm-run` just runs an npm script:

"This will start the Fable daemon and run an npm script in parallel (npm scripts are listed in the `package.json` file). In this case, `start` will run a Webpack development server."

> About using the `Fable daemon` term instead of server, see bellow.